prover/maths/common/polyext/poly.go

```diff
@@ -12,3 +14,33 @@ func EvalUnivariateMixed(pol []fext.GenericFieldElem, x fext.GenericFieldElem) f
 	}
 	return res
 }
+
+func MulByElement(a []fext.Element, b []field.Element) (res []fext.Element) {
```

Comment:
Doc

Reply:
// MulByElement multiplies the polynomials a and b in coefficient form and return the result in coefficient form also